internal/controller/replication.storage/volumereplication_controller.go

```diff
@@ -449,6 +449,11 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 					instance.Status.LastSyncBytes = &tb
 				}
 			}
+			msg := info.StatusMessage
+			if msg == "" {
+				msg = "message not provided by vendor"
```

Comment:
Let keep it empty if its not set by csi driver.

Reply:
I think we should leave the behavior as it is, we just advertise what is been provider by the csi driver, i think message is optional one and omit empty, if nothing is set, we will skip it. If they dont see the value in the status its the same assumption that csi driver doesnt provided any value to disable to the user, may be thats what they want convey to the users.